### ISSUE TITLE
Getting Errors

### ISSUE THREAD

**Issue by AceFire6** [willemt/py2graphql#1]
Hi @willemt 

Thank you for the great package 😄 

It's the best GQL client for python I've found!

I was wondering if there was a way to get the errors returned by a Graphql endpoint? I've looked at some of the source and can't see if you allow for that anywhere.

I don't mind making a PR if it's not currently supported and you're willing to consider it?

Thanks 😃

**Comment by willemt** [comment]
hi @AceFire6 
Thanks a lot, glad you like this client! 

When I get some more time, I was imaging that the client would simply just throw an `QueryException` when there's an error. Might be good to wrap the error dict inside an `addict.Dict()`. Throwing exceptions on errors is pythonic. You can see the beginnings of this on https://github.com/willemt/py2graphql/blob/master/py2graphl/core.py#L96

Btw, just please be aware of https://github.com/profusion/sgqlc. This repo has a similar approach, seems more active than mine and might have thought out the error handling system already.

**Comment by AceFire6** [comment]
Yeah, raising the exception definitely seems more Pythonic!

I had found `sgqlc` initially but I don't need the schema functionality. It seemed too big for what I needed.

This project seemed perfect because it left the deserialisation of the fields to me and the setup was super easy.

This project is also really intuitive. I managed to do things I thought should work even though they aren't documented 👍 

If you're not opposed to it, I'll make a PR to add basic error handling as you described.
